EditorVersion.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ue.4.26.win",
+    "name": "ue.4.27.win",
     "package": "26.2.0",
```

Comment:
The version will need to be updated to `"package": "27.0.1"` for CI to use the official release of 4.27!

Reply:
Oh thanks! I'll do that now.